lib/AST/ConformanceLookup.cpp

```diff
@@ -853,7 +853,7 @@ bool TypeBase::isSendableType() {
     return fas->isSendable();
 
   auto conformance = proto->getParentModule()->checkConformance(this, proto);
-  if (conformance.isInvalid())
+  if (conformance.isInvalid() || conformance.hasUnavailableConformance())
```

Comment:
`ProtocolConformanceRef` has `bool` operator that check `!isInvalid` inside so you can simply check for `!conformance` here.

I think you can simply everything down to `return conformance && !conformance.hasUnavailableConformance();` if you set `allowMissing` parameter of `checkConformance` to `false` in this case.